haskell/private/cabal_wrapper.sh.tpl

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# cabal_wrapper.sh <PKG_NAME> <SETUP_PATH> <PKG_DIR> <PACKAGE_DB_PATH> [SETUP_ARGS...]
+
+set -euo pipefail
+execroot=$(pwd)
+
+# Set these variables if unset.
+: ${LD_LIBRARY_PATH:=}
+: ${LIBRARY_PATH:=}
+export LD_LIBRARY_PATH=$execroot/$LD_LIBRARY_PATH
```

Comment:
`LD_LIBRARY_PATH` and `LIBRARY_PATH` are colon separated lists of paths. This only prefixes the `$execroot` to the first entry in that list.

Reply:
This line should include the variable substitution that is happening a few lines below. Will fix.